hudi-sync/hudi-datahub-sync/src/main/java/org/apache/hudi/sync/datahub/DataHubTableProperties.java

```diff
@@ -86,8 +86,8 @@ private static void addUserDefinedProperties(Map<String, String> properties, Dat
   private static void addSparkRelatedProperties(Map<String, String> properties, DataHubSyncConfig config, HoodieTableMetadata tableMetadata) {
     Map<String, String> sparkProperties = SparkDataSourceTableUtils.getSparkTableProperties(
         config.getSplitStrings(META_SYNC_PARTITION_FIELDS),
-        config.getString(META_SYNC_SPARK_VERSION),
-        config.getInt(HIVE_SYNC_SCHEMA_STRING_LENGTH_THRESHOLD),
+        config.getStringOrDefault(META_SYNC_SPARK_VERSION),
+        config.getIntOrDefault(HIVE_SYNC_SCHEMA_STRING_LENGTH_THRESHOLD),
```

Comment:
I see ADB sync also uses `config.getString(META_SYNC_SPARK_VERSION)`.  Should that be fixed separately?
```
Map<String, String> sparkTableProperties = SparkDataSourceTableUtils.getSparkTableProperties(config.getSplitStrings(META_SYNC_PARTITION_FIELDS),
          config.getString(META_SYNC_SPARK_VERSION), config.getInt(ADB_SYNC_SCHEMA_STRING_LENGTH_THRESHOLD), schema);
```